lib/board_implementation/CFBoard.h

```diff
@@ -1,42 +1,55 @@
 #pragma once

```

Comment:
I'm not sure how this work? Don't we have to use `#ifndef` or something like that?